_pages/home.md

```diff
@@ -87,7 +87,14 @@ multicolumn_list:
     url: https://www.zotero.org/
 ---
 
-Visit our [Authors](/authors/) section to learn how to obtain and use our styles; [Publishers](/publishers/) if you are a publisher or journal editor with an interest in CSL; [Developers](/developers/) if you want to edit or create styles, are looking to support CSL in your own software, or wish to contribute to development of CSL itself; [Blog](/blog/) for big news updates; [About us](/contact/) to see who is behind this open source project; and [Contact](/contact/) if you’re looking for support or just want to say hi.
+This website is divided into several sections:
 
-Our crowdsourced repository offers over 8000 free CSL citation styles, and CSL is used by commercial and open source products alike, including:
+* [Authors](/authors/) - learn how to obtain and use our CSL citation styles
```

Comment:
how about "find and use" -- obtain sounds odd to me.

Reply:
Would have to look, but might work better to also bold the category. 
Also, I think these should be endashes (`&endash;`) not hyphens.